### PR TITLE
Update autonick.txt with fantasy Dwarf names, 210 single&180 composite

### PR DESCRIPTION
**Pull request by IvoryTower33**

### CODE REVIEW

**Review by IvoryTower33 — COMMENTED** [review]
fixed for repeats

File: data/dfhack-config/autonick.txt [hunk 1]
```diff
@@ -7,6 +7,333 @@
...
+Bifur
+Bofur
+Bombur
+Ori
+Gimli
+Thrain
+Thror
+Fundin
+Frerin
+Nain
```
Comment (myk002) [inline data/dfhack-config/autonick.txt:26]: This one's in here twice

**Review by myk002 — COMMENTED** [review]
except for one duplicate, looks good! could you also add a note to the `docs/changelog.txt` file describing the improvement?

File: data/dfhack-config/autonick.txt (hunk 1 above)
Comment (myk002) [inline data/dfhack-config/autonick.txt:26]:
```suggestion
```

**Review by myk002 — APPROVED** [review]

**Pull request merged**